src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/OrderlyServerTests.kt

```diff
@@ -97,6 +135,17 @@ class OrderlyServerTests
         )
     }
 
+    @Test
+    fun `throws error on failure`()
```

Comment:
I think this is a partial duplicate of `throws error on failure` which I guess we don't need any more. But are we missing a test that it *doesn't* throw if `.throwOnError()` not called?